kcidb/test_oo.py

```diff
@@ -203,3 +203,169 @@ def test_two_object(self):
             '>checkout#>build#>test#',
             {"test", "build", "checkout", "revision"}, 2
         )
+
+
+class KCIDBMore00ClientTestCase(kcidb.unittest.TestCase):
```

Comment:
I couldn't come up with variable names for the new class, so I just added *More*. That probably needs to be changed.

Reply:
Since all our next tests are about traversing the objects, we can call it something like `KCIDBOOTraversingTestCase`.